src/manager.js

```diff
@@ -97,26 +92,31 @@ AdManager.prototype.initGoogleTag = function() {
   this.targeting = global.TARGETING || TargetingPairs.getTargetingPairs(AdZone.forcedAdZone()).pageOptions;
 
   this.setPageTargeting();
-  if (this.options.amazonEnabled) {
-    this.initAmazonA9();
-  }
 
   this.initialized = true;
 
   this.loadAds();
 };
 
 /**
- * initializes A9
+ * Fetch Amazon A9/Matchbuy/APS bids
  *
  * @returns undefined
 */
-AdManager.prototype.initAmazonA9 = function() {
-  if (typeof amznads !== "undefined" && amznads.apiReady) {
-    this.amznads = amznads;
-    this.amznads.getAds(this.amazonId);
-    this.amznads.setTargetingForGPTAsync('amznslots');
-  }
+AdManager.prototype.fetchAmazonBids = function(elementId, gptSizes) {
+	
+	window.apstag.fetchBids({
+		slots: [{
+			slotID: elementId,
+			sizes: gptSizes
+		}],
+		timeout: 2e3
+	}, function (bids) {
+		// Your callback method, in this example it triggers the first DFP request for googletag's disableInitialLoad integration after bids have been set
+		window.headertag.cmd.push(function () {
```

Comment:
Thought for all of these references, we can still use `window.googletag` instead of `window.headertag`

Reply:
@spra85 I've switched this to auto select between headertag and googletag.  According to index, the best practice is to always do this within headertag, but the reality in production seems to be that headertag is not always ready.